Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -607,17 +608,18 @@ String AXTextMarkerRange::toString() const
 
         StringBuilder builder;
         for (; !it.atEnd(); it.advance()) {
+            RefPtr<Node> countedNode = it.node();
```

Comment:
We can do:

```
RefPtr node = it.node();
```

and use that below.

Reply:
No problem! And yes, exactly. 

```
RefPtr node;        // ERROR: need to specify the template argument.
                    // (Because the compiler has no way of deducing the type here)
RefPtr<Node> node;  // OK

RefPtr node = it.node();       // OK
RefPtr<Node> node = it.node()  // OK but the <Node> is not needed. We prefer to be concise.